src/pages/sermons/videoComp.tsx

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @next/next/no-img-element */
+import Link from "next/link";
+
+const VideoComp = () => {
+	return (
+		<div className="videos">
+			<p className="text-style" style={{ paddingLeft: "2rem" }}>
+				All Sermon Videos
+			</p>
+			<div className="videos-wrapper">
+				<div className="febuary">
+					<p className="febuary-txt">Febuary, 2020</p>
+					<div className="febuary-cont">
+						{sermonDatas.map((sermonData, id) => (
```

Comment:
When I click a single video, I want to be redirected to the single video page. So you should create a page like so `/pages/[slug].tsx` for the single sermon page